src/main/kotlin/org/jmailen/gradle/kotlinter/tasks/format/FormatWorkerRunnable.kt

```diff
@@ -45,8 +45,8 @@ class FormatWorkerRunnable @Inject constructor(
                     val formattedText = formatFunc.invoke(file, ruleSets) { error, corrected ->
                         val errorStr = "$relativePath:${error.line}:${error.col}: [${error.ruleId}] ${error.detail}"
                         val msg = when (corrected) {
-                            true -> "Format fixed > $errorStr"
-                            false -> "Format could not fix > $errorStr"
+                            true -> "Format fixed > $projectDirectory${File.separator}$errorStr"
```

Comment:
According to the source code: https://github.com/pinterest/ktlint/blob/master/ktlint-core/src/main/kotlin/com/pinterest/ktlint/core/KtLint.kt#L414 `true` never gets returned so I couldn't test that :/